src/docs/arch-block.dox

```diff
@@ -15,6 +15,35 @@ re-written, they are written to new locations in the file. The size of a block
 is a multiple of the allocation size which is set during creation of the
 associated WiredTiger data file see: WT_SESSION::create.
 
+@plantuml_start{block.png }
+@startuml{block.png}
+
+title Block Layout
+node {
+ json "Block" as J {
+    "Page Header [00-27]": {
+      "recno (column store)": "00-07",
+      "write_gen": "08-15",
+      "mem_size": "16-19",
+      "entries": "20-23",
```

Comment:
We mention those two in the code:
```
        uint32_t entries; /* 20-23: number of cells on page */
        uint32_t datalen; /* 20-23: overflow data length */
```

Reply:
good catch

Reply:
Not sure the best way to format this. What do you think of 
```
      "entries (standard pages)": "20-23",
      "datalen (overflow pages)": "20-23",
```
  or
```
	"entries (or datalen for overflow pages)": "20-23",
```

Reply:
I like the second one. The first one may make you think it's two different things and we put the wrong numbers.

Reply:
Fixed 6e000f7de1